File: front_workspace/html_workspace/JS/day0514/js/Bird.js
/*
모든 새들의 어버이격인 그냥 새를 정의
 */
class Bird{
    constructor(color, age){
        this.color=color;
        this.age=age;
        console.log("Bird의 생성자 호출됨")
        console.log("부모님의 color는 "+this.color);
        console.log("부모님의 나이는 "+this.age);
    }
    eat(){
        console.log("새가 먹이를 먹어요");
    }
    fly(){
        console.log("새가 날아요");
    }
}

File: back_workspace/java_workspace/before/project0513/src/test/Bird.java
package test;

public class Bird{
	String name="나는 새";
	int age=3;
	
	/**
	새를 잠 재우고 싶으면 이 메서드 호출
	*/
	public void sleep(){
		System.out.println("잠을 자요");
	}
	
	/**
	새에게 먹이를 주세요
	*/
		public void eat(){
		System.out.println("밥을 먹어요");
	}
}